generators/app/templates/_package.json

```diff
@@ -28,19 +28,13 @@
     "generator-jhipster": ">2.26.0"
   },
   "devDependencies": {
-    "gulp": "^3.9.0",
```

Comment:
Its used by the gulpfile generated on the project root while running this
generator.

The gulp file has tasks to automate publishing of module so that module
author can just do gulp deploy-major|minor|patch to do version bump based
on the major|minor|patch, git commit.git push, tag, release and npm publish
in one step instead of doing these manually checkout the gulp file, its
same as the gulpfile I added on this generator root as well, you can try it
out for your next patch release may be just do gulp deploy-patch

Thanks & Regards,
Deepu

On Fri, Jan 22, 2016 at 1:03 PM, Pascal Grimaud <notifications@github.com>
wrote:

> In generators/app/templates/_package.json
> <https://github.com/jhipster/generator-jhipster-module/pull/9#discussion_r50502381>
> :
>
> > @@ -28,19 +28,13 @@
> >      "generator-jhipster": ">2.26.0"
> >    },
> >    "devDependencies": {
> > -    "gulp": "^3.9.0",
>
> I didn't see any use.
> Maybe I miss something?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jhipster/generator-jhipster-module/pull/9/files#r50502381>
> .
>